CONTRIBUTING.md

```diff
@@ -1,130 +1,132 @@
 # Contributing guide
```

Comment:
Is it me or you reformatted the whole `CONTRIBUTING.md`? I don't think we want to do that.

Reply:
I agree, reformatting the whole of `CONTRIBUTING.md` is a terrible idea. :) When I looked in my IDE's source-control-view it was a minimal diff, so either it's sneakily not showing whitespace in its diff view (thanks for that, IDE), or it reformats as part of the actual commit process (extra-thanks for that, IDE). 

I'll start again from scratch and use a different editor. It does make me think it would be helpful if we had a canonical, shared, set of IDE settings, though. We have some shared settings but not - apparently - enough to prevent this kind of time-sink.

Reply:
I mean, with hindsight I should have also checked the diff on the PR, but I naively trusted my IDE. Sorry about that!